### PR TITLE
[Event Hubs Client] Track Two: Fourth Preview (Reference Fixes)

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The intent of these changes is to fix package/project references for other client libraries in the shared repository to keep a consistent state and avoid mingling project and package references between client libraries.

# Last Upstream Rebase

Friday, September 20, 2019 9:38am (PDT)

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/samples/Azure.Messaging.EventHubs.CheckpointStore.Blob.Samples.csproj [hunk 1]
```diff
@@ -11,20 +11,6 @@
...
-    <PackageReference Include="System.Net.WebSockets.Client" />
-    <PackageReference Include="System.ValueTuple" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" />
-  </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\src\Azure.Messaging.EventHubs.CheckpointStore.Blob.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Azure.Messaging.EventHubs\samples\Azure.Messaging.EventHubs.Samples.csproj" />
```
Comment (ShivangiReja) [inline sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/samples/Azure.Messaging.EventHubs.CheckpointStore.Blob.Samples.csproj]: Just curious, why are we giving EventHubs samples reference(`Azure.Messaging.EventHubs\samples\Azure.Messaging.EventHubs.Samples.csproj`) here?

**Review by kinelski — APPROVED** [review]
LGTM.

File: sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/samples/Azure.Messaging.EventHubs.CheckpointStore.Blob.Samples.csproj (hunk 1 above)
Comment (kinelski) [reply sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/samples/Azure.Messaging.EventHubs.CheckpointStore.Blob.Samples.csproj]: Good point. I don't think we need it.

Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/samples/Azure.Messaging.EventHubs.CheckpointStore.Blob.Samples.csproj]: We don't.   Shivangi and I were able to confirm.  Update coming once tests pass.

**Review by ShivangiReja — APPROVED** [review]

**Pull request merged**